website/content/docs/what-is-vault.mdx

```diff
@@ -18,8 +18,22 @@ difficult and platform-specific. Adding on key rolling, secure storage, and
 detailed audit logs is almost impossible without a custom solution. This is
 where Vault steps in.
 
-Examples work best to showcase Vault. Please see the
-[use cases](/docs/use-cases).
+### How does Vault work?
+
+![How Vault Works](/img/how-vault-works.png)
+
+Vault validates, authorizes, and provides clients (users, machines, apps) access to secrets or stored sensitive data. The core Vault worklflow consists of four stages:
+
+![Vault Workflow](/img/vault-workflow-diagram1.png)
+
+* **Authenticate:** Client (users, apps, systems) authenticate with Vault using their identity.
+* **Validation:** Vault validates the client against third-party trusted sources.
+* **Authorize**: Client is matched against the Vault security policy.  This policy is a set of rules that direct which API endpoints a client has access to in a construct, using a Vault token.
+* **Access**: Vault provides access to secrets, keys, and encryption capabilities based on policies associated with the client’s identity.
+
+### Why Vault?
+
+Most enterprises today have credentials sprawled across their organizations, such as passwords, API keys, credentials that are in plain text, app source code, config files, and more. Since these credentials live everywhere, the sprawl can lead enterprises to a state where it's difficult and daunting to really know who has access and authorization to what. Further, having credentials in plain text increases the potential for malicious attacks, both by internal and external attackers. Vault was designed with these challenges in mind by helping you take all these credentials and centralize them so they are defined in one location, thereby, reducing unwanted exposure to credentials. But, Vault takes it a few steps further by making sure clients (users, apps, systems) are authenticated and explicitly authorized to access resources while providing an audit trail that captures and preserves a history of client actions.
```

Comment:
This is a pretty daunting block of text that I can see a visitor just glossing over. Do you think its possible to edit it down and/or breaking it into a few shorter paragraphs? Maybe turn it into a list of challenges?

Reply:
In lieu of a 20-minute video, I opted for text, but I can see how it may be a bit overwhelming. Looks like Jeff made some suggestions regarding the content and even broke it up a bit so it's more palatable.